### ISSUE TITLE
Gradient flow of the failing model

### ISSUE THREAD

**Issue by kushalj001** [hitvoice/DrQA#33]
I am trying to reproduce this paper and I have referred to this repository. My training is not giving satisfactory results. The metrics are subpar. Upon investigation, I tried plotting the layer gradients and found this. I have used the function from this [thread](https://discuss.pytorch.org/t/check-gradient-flow-in-network/15063/10).
![image](https://user-images.githubusercontent.com/32245327/83559837-d9d84a80-a532-11ea-957e-0835213b9e5d.png)
From the figure above, it seems as if the middle layers are not learning anything.
What should I try doing in order to fix this?
